docs/url-examples.md

```diff
@@ -139,6 +139,195 @@ Additional information:
 * [How to import CSV data](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-import-csv-data)
 
 
+## /api/v1/labels
+
+**Get a list of label names
+
+Single:
+<div class="with-copy" markdown="1">
+
+```bash
+curl -G 'http://localhost:8428/prometheus/api/v1/labels'
+```
+
+</div>
+
+Cluster:
+<div class="with-copy" markdown="1">
+
+```bash
+curl -G 'http://<vmselect>:8481/select/0/prometheus/api/v1/labels'
+```
+
+</div>
+
+**Querying label values
+
+Single:
+<div class="with-copy" markdown="1">
+
+```bash
+curl -G 'http://localhost:8428/prometheus/api/v1/label/job/values'
+```
+
+</div>
+
+Cluster:
+<div class="with-copy" markdown="1">
+
+```bash
+curl -G 'http://<vmselect>:8481/select/0/prometheus/api/v1/label/job/values'
+```
+
+</div>
+
+Additional information: 
+* [Prometheus querying API usage](https://docs.victoriametrics.com/#prometheus-querying-api-usage)
+* [Querying label values](https://prometheus.io/docs/prometheus/latest/querying/api/#querying-label-values)
+* [Getting label names](https://prometheus.io/docs/prometheus/latest/querying/api/#getting-label-names)
+
+## /api/v1/query
+
+**Performs PromQL instant query
+
+Single:
+<div class="with-copy" markdown="1">
+
+```bash
+curl -G 'http://localhost:8428/prometheus/api/v1/query?query=vm_http_request_errors_total'
+```
+
+</div>
+
+Cluster:
+<div class="with-copy" markdown="1">
+
+```bash
+curl -G 'http://<vmselect>:8481/select/0/prometheus/api/v1/query?query=vm_http_request_errors_total'
+```
+
+</div>
+
```

Comment:
Would be also nice to show example with complex query which needs encoding. For example, how to execute the following query sum(increase(foo{status="bar"}[5m])) by (status) ?

Reply:
Agreed

Reply:
Added, thanks for your assistance!